src/backend/controllers/aliases.ts

```diff
@@ -14,10 +14,6 @@ class Aliases {
   public static async get(aliasName: string): Promise<Alias> {
     const alias = await Alias.get(aliasName);
 
-    if (!alias.id) {
```

Comment:
why this statement has been removed?

Reply:
Sane error thrown twice that why removed